app/views/pages/documentation.html.erb

```diff
@@ -39,23 +37,26 @@
       <li class="about_li"><strong>Archive-It Information</strong>: In order to work with web archive collections, you must enter your Archive-It account username and password. Your Archive-It credentials are encrypted and salted.</li>
     </ul>
-    <%= image_tag("AUK_accountinfo.png", alt: "AUK Account Credentials Log In Screenshot", class:"body_img")%>
+    <%= image_tag("AUK_account_info.png", alt: "AUK Account Credentials Log In Screenshot", class:"body_img")%>
+    <%= image_tag("AUK_account_info2.png", alt: "AUK Account Credentials Log In Screenshot", class:"body_img")%>
     <p class="about_p">Click <strong>Update</strong></p>
     <p class="about_p">You will be redirected back into the main space. A note at the top of the page will indicate that AUK is syncing with your Archive-It account. The length of time to ingest your WARC collections will depend on the size of your collections and the number of other users syncing their collections. This process could take a few minutes to a few hours. You will receive an email, to the address you provided for your AU Cloud account, when the collections are synced.</p>
     <%= image_tag("AUK_user.png", alt: "AUK Account Credentials Log In Screenshot", class:"body_img")%>
     <p class="about_note"><strong>Note</strong>: You may update/change your AU Cloud or Archive-It credentials at any time by clicking <strong>Update</strong>. If you add a new collection in Archive-It or change your Archive-It credentials, you can click <strong>Update</strong> and redo the above steps to re-sync.</p>
 
     <h3 class="about_h3">Collections</h3>
-    <p class="about_p">Once your collections have been synced, you can return to the main collections screen and start analysis. Let's take a look at the collection information available to us in the main space:</p>
+    <p class="about_p">Once your collections have been synced, you can return to the main collections screen and start analysis. You can sort collections by clicking on the arrows beside each header. Let's take a look at the collection information available to us in the main space:</p>
     <ul class="about_ul">
       <li class="about_li"><strong>Title</strong> - The name of each collection is pulled directly from your Archive-It account; each title provides the access point for analysis.</li>
+      <li class="about_li"><strong>Analyzed</strong>- This header identifies whether a collection has been analyzed in AUK. If yes, you will see at least one of the following; a network diagram, downloadable derivative files, and/or a domains frequency table once you enter the collection's dashboard.</li>
       <li class="about_li"><strong>Public</strong> - This field notes whether or not the collection is public, and if so, will be listed on an institution's Archive-It page.</li>
       <li class="about_li"><strong>Files</strong> - Indicates the number of (WARC) files in each collection.</li>
```

Comment:
Change ` number of (WARC) files in each collection` to 

`number of ARC or WARC files in each collection`